src/main/java/core/basesyntax/service/ParserService.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax.service;
+
+import core.basesyntax.exception.FruitException;
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class ParserService {
+    private static final String SEPARATOR = ",";
+    private static final int OPERATOR_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int AMOUNT_INDEX = 2;
+    private static final int COUNT_WORDS = 3;
+    private static final String ANNOTATION = "type,fruit,quantity";
+
+    public List<FruitTransaction> convertToListFruitTransaction(List<String> strings) {
+        if (strings == null) {
+            throw new FruitException("Incoming string list is null");
+        }
+        strings = clearAnnotationCsvFile(strings);
+        return strings.stream()
+                .map(this::getFromString)
+                .collect(Collectors.toList());
+    }
+
+    private void checkPartsTransaction(String[] partsTransaction) {
```

Comment:
let's order your private methods so that the code could be read from top to bottom